packages/blade/src/components/Box/types.ts

```diff
@@ -36,4 +36,5 @@ export type BoxProps = {
   transform?: CSSObject['transform'];
   background?: CSSObject['background'];
   zIndex?: CSSObject['zIndex'];
+  id?: string;
```

Comment:
Need to add this explicitly. Box as div on web has this by default but native does not and leads to failure in TS checks on native

Reply:
Better to do this explicitly anyways